tests/test_utils.py

```diff
@@ -0,0 +1,100 @@
+"""Test utils."""
+import io
+import os
+import json
+import pandas as pd
+import tempfile
+import unittest
+from bitarray import bitarray
+from clkhash.serialization import serialize_bitarray
+from clkhash.clk import generate_clk_from_csv
+from clkhash.schema import from_json_file
+from blocklib import generate_candidate_blocks
+from anonlinkclient.utils import deserialize_filters, generate_candidate_blocks_from_csv, combine_clks_blocks
+
+from tests import *
+
+
+class TestUtils(unittest.TestCase):
+
+    def test_deserialize_bitarray(self):
+        """Test deserialize bitarray."""
+        # create simple bitarray bloom filter
+        lst = [True, False, True, False] * 2
+        bf = bitarray(lst)
+        serialized_bf = serialize_bitarray(bf)
+        deserialized_bf = deserialize_filters([serialized_bf])
```

Comment:
Is there a `deserialize_filter` instead of `deserialize_filters` function that could be used/tested here?

Reply:
We only defined `deserialize_filters` in `utils.py`